src/data/openthaigpt_pretraining_data/dedup_minhashlsh.py

```diff
@@ -0,0 +1,125 @@
+from datasketch import MinHash, MinHashLSH
```

Comment:
Can you create a script that run `generate_minhashlsh ` and `remove_duplicated` from input as huggingface dataset

Reply:
done generate_minhashlsh from input as huggingface dataset , but remove_duplicated still have input as lsh_dict (result of generate_minhashlsh. However, it will output as huggingface dataset